youtube.com.js

```diff
@@ -11,23 +11,23 @@ const siteSpecificDetector = { // eslint-disable-line no-unused-vars
       // The last one is known to be one with the highest resolution
       thumbnail = thumbnail[thumbnail.length - 1]?.url
     }
-    return (typeof(thumbnail) === 'string' && (thumbnail.startsWith('/') || thumbnail.startsWith('http://') || thumbnail.startsWith('https://'))) 
-        ? thumbnail : null
+    return (typeof (thumbnail) === 'string' && (thumbnail.startsWith('/') || thumbnail.startsWith('http://') || thumbnail.startsWith('https://')))
```

Comment:
This is just an observation but I don't think we normally add the brackets around

```ts
// prefer
typeof thing === 'type'
// over
typeof (thing) === 'type'
```

But it's not in the style guide, and it's not your code so this is fine :laughing:

Reply:
Oh, yeah. I always forget it 🤣